public/static/locales/eng/terms.md

```diff
@@ -1,51 +1,51 @@
 
-Sem þátttakandi samþykkir þú að öll þín framlög (raddupptaka, og eftir atvikum lýðfræðilegar upplýsingar, yfirferðir og endurgjöf) falli undir [Creative Commons by attribution](https://creativecommons.org/licenses/by/4.0/).
+As a participant, you agree that all of your contributions (voice recordings, and, as appropriate, demographic information, reviews, and feedback) will be covered by the Creative Commons by attribution license [Commons by attribution license](https://creativecommons.org/licenses/by/4.0/).
 
-Til þess að taka þátt þarf að skrá inn lýðfræðilegar upplýsingar, þ.e. aldursbil, kyn og móðurmál. Með því að skrá inn lýðfræðilega upplýsingar um þig staðfestir þú að þær séu skráðar af heilindum og séu sannar. Ósannar og rangar upplýsingar geta haft neikvæð áhrif á gagnasöfnunina.
+In order to participate, you need to enter demographic information, i.e. age range, gender and mother tongue. By entering demographic information about yourself, you confirm that it is recorded with integrity and is true. False and incorrect information can adversely affect data collection.
 
-Til þess að ólögráða einstaklingar geti tekið þátt þarf leyfi forsjáraðila. Ef þátttakandi er ólögráða þarf hann að fá leyfi forsjáraðili fyrir þátttöku í gerð gagnasafnsins. Í því skyni þarf þátttakandi að gefa upp kennitölu og netfang forsjáraðila. Forsjáraðili fær í kjölfarið tölvupóst og er beðinn um að svara honum með því að smella á hlekk. Forsjáraðilinn veitir þar með samþykki fyrir þá kennitölu sem um ræðir fyrir þátttöku í Samrómi. Þessar upplýsingar verða ekki gefnar út með gagnasafninu og þær verða ekki geymdar lengur en þörf krefur.
+In order for minors to participate, a guardian's permit is required. If a participant is a minor, he or she must obtain permission from the custodian to participate in the creation of the database. To do this, the participant must provide his or her ID number and the e-mail address of the custodian. The custodian then receives an email and is asked to respond to it by clicking on a link. The custodian thereby gives consent for the ID number in question for participation in Samrómur. This information will not be published with the database and will not be stored longer than necessary.
 
-Þátttakendur geta valið að gefa upp upplýsingar sem segja til um undir hvaða formerkjum þeir taka þátt, s.s. nafn skóla, stofnunar eða fyrirtækis. Þessar upplýsingar eru einungis notaðar til að halda utan um þátttöku hjá hópum og kunna að vera birtar á vef síðunnar. Upplýsingarnar verða hvorki gefnar út með gagnasafninu né geymdar lengur en þörf krefur.
+Participants can choose to provide information indicating which school or company they participate for. This information is only used to keep track of group participation and competitions and may be published on the site's website. The information will not be published with the database or stored longer than necessary.
 
-## Aðgengi að gagnasafni
+## Access to a database
```

Comment:
Should this be access to **the** database? as in the samromur one? not **a** database

Reply:
Or just Access to database

Reply:
ok